File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.py
import sys

n, k = map(int, sys.stdin.readline().split())

def fac(num):
  sum = 1
  for i in range(1, num + 1):
    sum *= i

  return sum

bin_coe = fac(n) // (fac(k) * fac(n - k))
print(bin_coe)

File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.cc
#include <iostream>
using namespace std;

int fac(int num) {
  int sum = 1;

  for (int i = 1; i < num + 1; i++)
    sum *= i;

  return sum;
}

int main() {
  int n, k;
  cin >> n >> k;

  int bin_coe = fac(n) / (fac(k) * fac(n - k));
  cout << bin_coe << "\n";
}